deepfence_worker/cronjobs/cloud_compliance.go

```diff
@@ -320,12 +330,7 @@ func CachePostureProviders(msg *message.Message) error {
 	if err != nil {
 		return err
 	}
-
-	rdb, err := directory.RedisClient(ctx)
-	if err != nil {
-		return err
-	}
-	err = rdb.Set(ctx, constants.RedisKeyPostureProviders, string(postureProvidersJson), 0).Err()
+	err = rdb.Set(ctx, constants.RedisKeyPostureProviders, string(postureProvidersJson), time.Hour*2).Err()
```

Comment:
Given that this job is executed on a regular basis, we don't need to set an expiry on those entries.
If something needs to be cleared up, let's do it in that job.

Reply:
Already fixed in main

Reply:
Just realized it's part of the rebase 👍